Content.Tests/DMProject/Tests/Image/appearance.dm

```diff
@@ -0,0 +1,10 @@
+/obj/thingtocopy
```

Comment:
Using atom-derivatives in a unit test is generally not a good idea. Our unit testing framework doesn't start up a whole server, and so the entity systems/entities responsible for atoms aren't created. This is the reason tests are failing on this PR.

If you really need to test atom functionality, you need to add to the `Content.IntegrationTests` project.

Reply:
Okay the test probably isn't all that necessary, but I'll move it over to integration tests just to learn how to do that 👍